pages/_app.js

```diff
@@ -9,6 +9,11 @@ export default function App({ Component, pageProps }) {
   //build a useState for the result of the Form
   const [formResults, setFormResults] = useState(0);
   const [randomSurprise, setRandomSurprise] = useState(0);
+  const [activePage, setActivePage] = useState(false);
```

Comment:
If this would be called [isPageActive , setIsPageActive] it would be more clear what this is about.
Reading "activePage" I would expect to see a string which page is the active page.